pkg/cli/start.go

```diff
@@ -479,6 +479,7 @@ func runStart(cmd *cobra.Command, args []string) error {
 	case sig := <-signalCh:
 		returnErr = fmt.Errorf("received signal '%s' during shutdown, initiating hard shutdown", sig)
 		log.Errorf(shutdownCtx, "%v", returnErr)
+		fmt.Fprintf(os.Stdout, "%v\n", returnErr)
```

Comment:
We use `fmt.Fprint{f,ln}(os.Stdout` elsewhere in this file. I'm matching that convention here. Switched to `Fprintln`.